Python/compile.c

```diff
@@ -140,6 +140,18 @@ location_is_after(location loc1, location loc2) {
              (loc1.col_offset > loc2.end_col_offset));
 }
 
+static inline bool
+same_location(location a, location b)
+{
+    if (a.lineno < 0 && b.lineno < 0) {
```

Comment:
I’d check also that they are the same value in case we will want to distinguish between ‘no location’ and ‘unknown location’ or something like that. 

(I’m assuming you don’t rely on the full equality check because we’re inconsistent about what the other 3 values are).

Reply:
Should an "unknown location" ever exist? That sounds like a compiler bug.

Reply:
> (I’m assuming you don’t rely on the full equality check because we’re inconsistent about what the other 3 values are).

Actually, it appears that we can remove this special case entirely. Just using the full equality check results in the exact same PYC sizes for the entire stdlib.